### PR TITLE
Feature/new committee page

### PR DESCRIPTION
**Pull request by noahmanger**
This implements several features of the new committee pages. It doesn't include the new mini charts and such, but does organize the existing data in the new layout. 

- [Financial summary](https://cloud.githubusercontent.com/assets/1696495/22846961/12c2665c-ef9f-11e6-8d9a-2f7ef0665d00.png)
- [Filings](https://cloud.githubusercontent.com/assets/1696495/22846977/25f57dc2-ef9f-11e6-9755-948b1e2d0094.png)
- [About](https://cloud.githubusercontent.com/assets/1696495/22846992/3f0f3960-ef9f-11e6-9048-741b8ef3ba73.png)
- [Individual contributions](https://cloud.githubusercontent.com/assets/1696495/22847010/51b40d5c-ef9f-11e6-89d0-4cd99fa51f3c.png)

To do:
- [x] Verify ordering of filings tables is correct
- [x] Double check all fields in financial summary table

Resolves https://github.com/18F/openFEC-web-app/issues/1831
Resolves https://github.com/18F/openFEC-web-app/issues/1833
Resolves https://github.com/18F/openFEC-web-app/issues/1832
Resolves https://github.com/18F/openFEC-web-app/issues/1834

### CODE REVIEW

**Comment by noahmanger** [comment]
Weird! I don't have that problem. Have you pulled down the latest (and latest fec-style?)

**Comment by noahmanger** [comment]
@xtine nevermind! I have that problem too. Looking into it.

**Comment by xtine** [comment]
Side nav links working now 👍

File: static/js/pages/committee-single.js [hunk 1]
```diff
@@ -281,14 +267,39 @@ var aggregateCallbacks = {
...
 $(document).ready(function() {
   // Set up data tables
   $('.data-table').each(function(index, table) {
     var $table = $(table);
     var committeeId = $table.attr('data-committee');
     var cycle = $table.attr('data-cycle');
     var query = {cycle: cycle};
-    var path;
+    var path,
+        opts;
```
Comment (xtine) [inline static/js/pages/committee-single.js]: Can opts be declared separately for consistency? `var opts;`

**Pull request merged**